akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala

```diff
@@ -252,7 +252,7 @@ private[http] final class BodyPartParser(defaultContentType: ContentType,
 
 private[http] object BodyPartParser {
   // http://tools.ietf.org/html/rfc2046#section-5.1.1
-  val boundaryCharNoSpace = CharPredicate.Digit ++ CharPredicate.Alpha ++ "'()+_,-./:=?"
+  val boundaryChar = CharPredicate.Digit ++ CharPredicate.Alpha ++ "'()+_,-./:=? "
```

Comment:
it's exclusively used [here](https://github.com/akka/akka/pull/18903/files#diff-1e81ca1591fa3a9265369f2153fa2f93R35)